test/integration/application-log-assersion/webapp/test/GherkinTestRunner.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Using Gherkin with OPA5</title>
+
+    <script
+      id="sap-ui-bootstrap" src="https://sapui5.hana.ondemand.com/resources/sap-ui-core.js"
```

Comment:
Please use the local UI5 bootstrap instead
```suggestion
      id="sap-ui-bootstrap" src="../resources/sap-ui-core.js"
```